product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2701 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Basic Tour"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "* This tour will introduce you to Neptune client API. In the next few steps you will see how to create experiment in the project and log meaningful training data to the running experiment.\n",
+    "* This tour use popular deep learning libraries TensorFlow and Keras to make it looks familiar."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 1: Import Neptune and TensorFlow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Select Neptune project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "* Use ``neptune.init`` to set project. It's necessary to tell Neptune to which project you want to log experiments' data.\n",
+    "* Consider using your own project and API token. Here is how to:\n",
+    "  * [get api token](https://docs.neptune.ai/security-and-privacy/api-tokens/how-to-find-and-set-neptune-api-token.html)\n",
+    "  * [create project](https://docs.neptune.ai/workspace-project-and-user-management/projects/create-project.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.init('shared/tour-with-tf-keras',\n",
+    "             api_token='ANONYMOUS')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 3: Create Neptune experiment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "* Use ``neptune.create_experiment`` to make new experiment in the project.\n",
+    "* Use returned Experiment object ``exp`` to log relevant data during experiment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "exp = neptune.create_experiment(name='keras-training-basic')"
```

Comment:
This is/could be one of the first contacts with Neptune. 
Using explicit `exp` is a bit more advanced and I strongly feel that we shouldn't use/show it here. 
Could you drop it?

I presume the reason why not is that later it's hard to test. 
But you can have 3 cells

```python
# tests
exp = neptune.get_experiment()
```

```python
# code

neptune.stop()
```

```python
# tests

all the tests go here
```

I did that in the tensorflow-keras examples and it worked nicely.

Reply:
dropped.